netcore/NetCore.Test/CacheFactoryUnitTest.cs

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+using Apache.Geode.Client;
+using Xunit;
+
+namespace GeodeDotNetTest {
+  [Collection("Geode .net Core Collection")]
+  public class CacheFactoryUnitTests {
```

Comment:
The naming convention used by the Java and C++ sources is `ClassTest(s)` where `Class` is the class under testing and `Test` or `Tests` (not consistent) is used to denote unit tests. Unit tests are also isolated away from other forms of longer feedback tests, like integration, also suffixing them `UnitTest` was deemed redundant.

Reply:
Everything is now singular `Test`, and I removed `Unit` from all the class names and renamed the files appropriately.